lib/parsers/webhook_json_parser.rb

```diff
@@ -100,13 +101,11 @@ def deleted?
       end
 
       def repo_name
-        case @vcs
-        when 'github'
-          @data['repository']['name']
-        when 'gitlab'
+        if @vcs == 'gitlab'
           @data['project']['name']
+        else
```

Comment:
I think I'd prefer `when 'github', 'stash', 'bitbucket'` or whatever.  It makes it much clearer IMO.

Reply:
Tested an explicit check with a LOGGER.error call on else and realized that it would never get this far is the payload didn't match one of github, gitlab, bitbucket, stash, or TFS as the `detect_vcs` method already checks for that and raises a `StandardError` if there is no match